satellite/audit/service.go

```diff
@@ -63,13 +78,32 @@ func (service *Service) Run(ctx context.Context) (err error) {
 	defer mon.Task()(&ctx)(&err)
 	service.log.Info("Audit cron is starting up")
 
-	return service.Loop.Run(ctx, func(ctx context.Context) error {
-		err := service.process(ctx)
+	group, ctx := errgroup.WithContext(ctx)
+
+	service.Loop.Start(ctx, group, func(ctx context.Context) (err error) {
```

Comment:
Do we need the entire old service loop?  We probably need reverify, but wasn't the old cursor loop going away?